config/filter.d/gitea-auth.conf

```diff
@@ -0,0 +1,3 @@
+[Definition]
+failregex = .*Failed authentication attempt.*remote_addr=.*$
```

Comment:
The RE doesn't contain `<HOST>` or `<ADDR>`
Beside that such REs cannot be accepted - they are vulnerable because unanchored, has an interim catch-all, etc.